class.jetpack.php

```diff
@@ -5956,14 +5956,16 @@ public static function dns_prefetch( $new_urls = null ) {
 	public function wp_dashboard_setup() {
 		if ( self::is_active() ) {
 			add_action( 'jetpack_dashboard_widget', array( __CLASS__, 'dashboard_widget_footer' ), 999 );
+			$widget_title = __( "Jetpack", 'jetpack' );
```

Comment:
translation function uses both double quotes and single quotes.  Let's just go single quotes.